libexec/scoop-search.ps1

```diff
@@ -11,6 +11,11 @@ param($query)
 
 $list = [System.Collections.Generic.List[PSCustomObject]]::new()
 
+if(!$query -or $query -in @($null, '-h', '--help', '/?')) {
```

Comment:
Uh.. this is a breaking change 😅

`scoop search` without arguments has been used to list all local manifests, facilitating commands like `scoop search | ? source -eq extras | measure`.

Reply:
Oh, yeah for pipelines. okay reverted 😄